Sources/AsyncHTTPClient/HTTPClient+HTTPCookie.swift

```diff
@@ -61,6 +61,10 @@ extension HTTPClient {
             self.name = nameAndValue[0]
             self.value = nameAndValue[1]
 
+            guard !self.name.isEmpty else {
+                return nil
+            }
```

Comment:
This new guard does not seem to be met by an associated test: mind adding one?

Reply:
we do have them:
```
XCTAssertNil(HTTPClient.Cookie(header: "=;", defaultDomain: "exampe.org"))
```
should hit it

Reply:
:+1: